docs/walkthrough.md

```diff
@@ -0,0 +1,204 @@
+
+    Inferences/Second vs. Client Average Batch Latency
+    Concurrency: 1, throughput: 482.8 infer/sec, latency 12613 usec
+    ```
+
+    - The defintion of "performing well" is subject to change for each use case. There are many variables that can be tweaked just within your Triton configuration (`config.pbtxt`) to obtain different results.
+
+3. How can I improve my model performance?
+    - To further understand the best model configuration you can provide to Triton for your use case, Triton's [Model Analyzer](https://github.com/triton-inference-server/model_analyzer) tool can help. After running Model Analyzer to find the optimal configurations for your model/use case, you can transfer the generated config files to your [Model Repository](https://github.com/triton-inference-server/server/blob/main/docs/model_repository.md). Model Analyzer provides a [Quickstart](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/quick_start.md) guide with some examples to walk through.
+    - Upon serving the model with the newly optimized configuration file found by Model Analyzer and running Perf Analyzer again, you should expect to find better performance numbers in many cases compared to a default config.
+    - Some parameters that can be tuned for a model may not be exposed to Model Analyzer since they don't apply to all models. For instance, [backends](https://github.com/triton-inference-server/backend) can expose backend-specific configuration options that can be tuned as well. The [ONNXRuntime Backend](https://github.com/triton-inference-server/onnxruntime_backend) for example has several [parameters](https://github.com/triton-inference-server/onnxruntime_backend#model-config-options) that affect the level of parallelization used when executing inference on a model and may be worth investigating if the defaults are not providing sufficient performance. 
+    - For manual tuning of your model configuration, there is further information in our [Optimization](https://github.com/triton-inference-server/server/blob/main/docs/optimization.md) docs.
+
+## Other Areas of Interest
+
+1. My model performs slowly when it is first loaded by Triton (cold-start penalty)
+    - Triton exposes the ability to run [ModelWarmup](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#model-warmup) requests when first loading the model to ensure that the model is sufficiently initialized before being marked "READY" for inference.
+
+2. Why doesn't my model perform significantly faster on GPU?
+    - Most official backends supported by Triton are optimized for GPU inference and should perform well on GPU out of the box.
+    - However, Triton expose options to further optimize your model for GPU execution for you. Triton's [Framework Specific Optimizations](https://github.com/triton-inference-server/server/blob/main/docs/optimization.md#framework-specific-optimization) goes into further detail on this topic.
```

Comment:
>expose

Change to "exposes".

>  However, Triton expose options to further optimize your model for GPU execution for you.

Even with the above change, this sentence reads a bit awkward. Perhaps, "Triton exposes options for you to optimize your model further on the GPU." Drop the "However," bullet points don't need adverbs as you are declaring a list rather than splitting paragraphs into points.